solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -140,54 +140,54 @@ public List<Option> getOptions() {
             .build(),
         Option.builder()
             .longOpt("example-dir")
-            .argName("DIR")
             .hasArg()
+            .argName("DIR")
             .required(false)
             .desc(
                 "Path to the Solr example directory; if not provided, ${serverDir}/../example is expected to exist.")
             .build(),
         Option.builder()
             .longOpt("url-scheme")
-            .argName("SCHEME")
             .hasArg()
+            .argName("SCHEME")
             .required(false)
             .desc("Solr URL scheme: http or https, defaults to http if not specified.")
             .build(),
         Option.builder("p")
-            .argName("PORT")
+            .longOpt("port")
             .hasArg()
+            .argName("PORT")
             .required(false)
             .desc("Specify the port to start the Solr HTTP listener on; default is 8983.")
-            .longOpt("port")
             .build(),
         Option.builder()
-            .argName("HOSTNAME")
+            .longOpt("host")
             .hasArg()
+            .argName("HOSTNAME")
             .required(false)
             .desc("Specify the hostname for this Solr instance.")
-            .longOpt("host")
             .build(),
         Option.builder("c")
+            .longOpt("cloud")
             .required(false)
             .desc(
                 "Start Solr in SolrCloud mode; if -z not supplied, an embedded ZooKeeper instance is started on Solr port+1000, such as 9983 if Solr is bound to 8983.")
-            .longOpt("cloud")
             .build(),
         Option.builder("m")
-            .argName("MEM")
+            .longOpt("memory")
             .hasArg()
+            .argName("MEM")
             .required(false)
             .desc(
                 "Sets the min (-Xms) and max (-Xmx) heap size for the JVM, such as: -m 4g results in: -Xms4g -Xmx4g; by default, this script sets the heap size to 512m.")
-            .longOpt("memory")
             .build(),
-        Option.builder("a")
-            .argName("OPTS")
```

Comment:
I see your question now..   So the thing that is unqiue about RunExampleTool compared to our other Java `*Tool` is that the arguments passed to it are NOT the ones typed by the human user!  Instead, the `start` command parses all the args and creates a NEW set of commands to RunExampleTool.   So we can change how RunExampleTool works as LONG as we maps the args in the `start` command.

Reply:
That makes sense. I was just unsure how changes in public classes should be treated in general, so I assumed it would also require deprecation for cases where someone accesses and uses the class(es) directly.